src/XCCDF/xccdf_session.c

```diff
@@ -1554,7 +1554,11 @@ int xccdf_session_remediate(struct xccdf_session *session)
 	xccdf_result_set_version(session->xccdf.result,
 			benchmark != NULL ? xccdf_benchmark_get_version(benchmark) : NULL);
 	xccdf_result_fill_sysinfo(session->xccdf.result);
-	return xccdf_policy_remediate(xccdf_session_get_xccdf_policy(session), session->xccdf.result);
+
+	if ((res = xccdf_policy_remediate(xccdf_session_get_xccdf_policy(session), session->xccdf.result)) != 0)
```

Comment:
Why do we even have this function as int return if all it does it run remediate functions and then return 0? @mpreisler shouldn't we make this void return? As this *if* will never be true, I don't think it's really needed.